src/WebView.android.tsx

```diff
@@ -119,6 +119,8 @@ const WebViewComponent = forwardRef<{}, AndroidWebViewProps>(({
     clearFormData: () => Commands.clearFormData(webViewRef.current),
     clearCache: (includeDiskFiles: boolean) => Commands.clearCache(webViewRef.current, includeDiskFiles),
     clearHistory: () => Commands.clearHistory(webViewRef.current),
+    // @ts-ignore
+    getNativeTag: () => webViewRef.current._nativeTag,
```

Comment:
It would be nice to have comments about this... e.g., what this is and why you expect it to be there and expose it even though it's not part of the type (and appears undocumented)...

And, along those lines, if it's going to be exposed then it probably should be added to `react-native-webview`'s own types/documentation.

Reply:
You are right, it's undocumented. So, maybe this would be better instead:

```js
getNativeWebView: () => webViewRef.current
```

or

```js
getNativeWebViewRef: () => webViewRef
```

Which does look better for you?
This way, we no longer need `'no-underscore-dangle': 'off'`.

Reply:
That seems okay, but I still think it should be added to the types and documented (if only to discourage its use except for advanced scenarios)